db/migrate/20170124095318_create_complaints.rb

```diff
@@ -23,34 +23,6 @@ def change
       t.integer :ctbfc, default: 0
       t.integer :cobfc, default: 0
 
-      t.integer :complaints_facility_code, default: 0
-      t.integer :complaints_financial_by_fac_code, default: 0
-      t.integer :complaints_quality_by_fac_code, default: 0
-      t.integer :complaints_refund_by_fac_code, default: 0
-      t.integer :complaints_marketing_by_fac_code, default: 0
-      t.integer :complaints_accreditation_by_fac_code, default: 0
-      t.integer :complaints_degree_requirements_by_fac_code, default: 0
-      t.integer :complaints_student_loans_by_fac_code, default: 0
-      t.integer :complaints_grades_by_fac_code, default: 0
-      t.integer :complaints_credit_transfer_by_fac_code, default: 0
-      t.integer :complaints_job_by_fac_code, default: 0
-      t.integer :complaints_transcript_by_fac_code, default: 0
-      t.integer :complaints_other_by_fac_code, default: 0
-
-      t.integer :complaints_main_campus_roll_up, default: 0
-      t.integer :complaints_financial_by_ope_id_do_not_sum, default: 0
-      t.integer :complaints_quality_by_ope_id_do_not_sum, default: 0
-      t.integer :complaints_refund_by_ope_id_do_not_sum, default: 0
-      t.integer :complaints_marketing_by_ope_id_do_not_sum, default: 0
-      t.integer :complaints_accreditation_by_ope_id_do_not_sum, default: 0
-      t.integer :complaints_degree_requirements_by_ope_id_do_not_sum, default: 0
-      t.integer :complaints_student_loans_by_ope_id_do_not_sum, default: 0
-      t.integer :complaints_grades_by_ope_id_do_not_sum, default: 0
-      t.integer :complaints_credit_transfer_by_ope_id_do_not_sum, default: 0
-      t.integer :complaints_jobs_by_ope_id_do_not_sum, default: 0
-      t.integer :complaints_transcript_by_ope_id_do_not_sum, default: 0
-      t.integer :complaints_other_by_ope_id_do_not_sum, default: 0
-
```

Comment:
After this gets merged I think it's important we stop updating migrations in place and start producing actual migrations. It makes reviewing schema changes easier and creates fewer problems updating our local dev environments.